content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -163,16 +163,21 @@ There are a few limitations with `env` at the moment:
 
 For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files. The Prisma CLI expects to find the `.env` file:
 
-1. Next to your schema file
-2. In the `./prisma` folder, if it is not found next to your schema file
+1. In the root folder of your project (`./.env`)
+1. From the location specified by the `--schema` argument
```

Comment:
That is technically not fully correct. The argument points to the `schema.prisma` file, and then the CLI looks in the same folder of that file for a `.env` file afaik.